axelor-account/src/main/java/com/axelor/apps/account/service/custom/AccountingReportValueAbstractService.java

```diff
@@ -198,7 +216,7 @@ protected int getColumnStyleSelect(
 
   protected List<String> getAccountFilters(
       Set<AccountType> accountTypeSet, String groupColumnFilter, boolean areAllAccountSetsEmpty) {
-    List<String> queryList = Arrays.asList("self.isRegulatoryAccount IS FALSE");
+    List<String> queryList = new ArrayList<>(Arrays.asList("self.isRegulatoryAccount IS FALSE"));
```

Comment:
Hum why do you need to add this? It's exactly what is done in Arrays.asList

Reply:
As stated in java doc, `Arrays.asList` will `Returns a fixed-size list backed by the specified array.`, so I need this to add new elements